app/src/main/java/eu/kanade/presentation/track/TrackerSearchPreviewProvider.kt

```diff
@@ -88,8 +93,10 @@ internal class TrackerSearchPreviewProvider : PreviewParameterProvider<@Composab
         it.finished_reading_date = 0L
         it.tracking_url = "https://example.com/tracker-example"
         it.cover_url = "https://example.com/cover.png"
-        it.start_date = Instant.now().minus((1L..365).random(), ChronoUnit.DAYS).toString()
+        it.start_date = formatter.format(Date.from(Instant.now().minus((1L..365).random(), ChronoUnit.DAYS)))
         it.summary = lorem((0..40).random()).joinToString()
+        it.publishing_status = if ((0..1).random() == 1) "Finished" else ""
+        it.publishing_type = if ((0..1).random() == 1) "Oneshot" else ""
```

Comment:
And for publication status
```suggestion
        it.publishing_type = if (Random.nextBoolean()) "Oneshot" else ""
```